templates/python/.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+---
+repos:
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ["--settings", "setup.cfg"]
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
```

Comment:
before we merge this, could you go through this list of tools and update each of them to the latest version? for example Black is already at 22.12.0, so we shouldn't use this old version for new projects.
Then the same version needs to be applied in `pyproject.toml`. I know it's a bit tedious but it would be great to get this in :)

Reply:
On it

Reply:
done

Reply:
perfect, thanks!